messaging.rst

```diff
@@ -14,22 +14,16 @@ Data Structures
 Offchain Balance Proof
 ----------------------
 
-Data required by the smart contracts to update the payment channel end of the participant that signed the balance proof.
-Messages into smart contracts contain a shorter form called :ref:`Onchain Balance Proof <balance-proof-onchain>`.
+An offchain balance proof is a set of fields of Raiden protocol messages.  The fields contain enough information for producing an :ref:`Onchain Balance Proof <balance-proof-onchain>`, which is required by the smart contracts to update the payment channel (especially, the state of the participant that signed the balance proof).  The offchain balance contains more information for easier bookkeeping.
 
-The signature must be valid and is defined as:
 
-::
-
-    ecdsa_recoverable(privkey, keccak256(balance_hash || nonce || additional_hash || channel_identifier || token_network_address || chain_id)
-
-Fields
-^^^^^^
+Fields of the Offchain Balance Proof message
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
 +--------------------------+------------+--------------------------------------------------------------------------------+
 | Field Name               | Field Type |  Description                                                                   |
 +==========================+============+================================================================================+
-|  nonce                   | uint256    | Strictly monotonic value used to order transfers. The nonce starts at 1        |
+|  nonce                   | uint256    | Strictly monotonic value used to order messages. The nonce starts at 1         |
```

Comment:
Now I'm not sure after looking into the code.  In the code, nonces are on balance proofs not on messages.

Reply:
I agree that `transfers` is not optimal, as the `nonce` is increased multiple times during a `transfer`. I am fine with `messages` or even `balance proofs`.

Reply:
I'll go for `balance proofs`.

Reply:
Done!